<testsuite name="specs.cadastro.Cadastro no site de teste" tests="1" errors="1" failures="0" skipped="0" time="10.035119" timestamp="2022-07-27T15:10:46.836849" hostname="CTNDELLGMJCP63"><testcase classname="specs.cadastro.Cadastro no site de teste" name="Realizar cadastro" status="failed" time="10.035119"><error type="TimeoutError" message="Timeout 10000ms exceeded.&#10;=========================== logs ===========================&#10;waiting for selector &quot;#navbarSupportedContent &gt; ul &gt; li:nth-child(2) &gt; a&quot;&#10;============================================================">
<![CDATA[
Failing step: Dado que acesse o formulario de cadastro ... failed in 10.035s
Location: features/specs/cadastro.feature:7
Traceback (most recent call last):
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\behave\model.py", line 1329, in run
    match.run(runner.context)
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\behave\matchers.py", line 98, in run
    self.func(context, *args, **kwargs)
  File "features\steps\cadastro_steps.py", line 6, in acessar_pagina_cadastro
    context.react.home.acessar_registro(context)
  File "C:\python_with_playwright\bdd\features\page_objects\React\pages\home.py", line 28, in acessar_registro
    context.page.click(body_elements['MENU_REGISTRO'])
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\playwright\sync_api\_generated.py", line 8007, in click
    self._sync(
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\playwright\_impl\_sync_base.py", line 89, in _sync
    return task.result()
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\playwright\_impl\_page.py", line 691, in click
    return await self._main_frame.click(**locals_to_params(locals()))
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\playwright\_impl\_frame.py", line 474, in click
    await self._channel.send("click", locals_to_params(locals()))
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\playwright\_impl\_connection.py", line 43, in send
    return await self._connection.wrap_api_call(
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\playwright\_impl\_connection.py", line 370, in _
    return await result
  File "C:\Users\vinicius.jacques\AppData\Local\Programs\Python\Python310\lib\site-packages\playwright\_impl\_connection.py", line 78, in inner_send
    result = next(iter(done)).result()
playwright._impl._api_types.TimeoutError: Timeout 10000ms exceeded.
=========================== logs ===========================
waiting for selector "#navbarSupportedContent > ul > li:nth-child(2) > a"
============================================================
]]>
</error><system-out>
<![CDATA[
@scenario.begin
  Cenário: Realizar cadastro
    Dado que acesse o formulario de cadastro ... failed in 10.035s
    Quando preencher as informações necessárias ... skipped in 0.000s
    Então o usuario deve ser cadastrado ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase></testsuite>